src/main/index.ts

```diff
@@ -102,7 +113,7 @@ function createMainWindow() {
 
 // quit application when all windows are closed
 app.on("window-all-closed", () => {
-  app.quit();
+  //app.quit();
```

Comment:
Shouldn't this also have a `!isMac` check?

Reply:
This was branched from the main branch - the macOS QoL PR handles implementing window reopening, so I felt no need to deal with it here. Less moving parts.